stable-swap-client/src/instruction.rs

```diff
@@ -632,11 +654,15 @@ pub fn initialize(
     nonce: u8,
     amp_factor: u64,
     fees: Fees,
+    token_a_exchange_rate_override: Fraction,
+    token_b_exchange_rate_override: Fraction,
```

Comment:
Let's follow up and revert the override setters in `init`. Instead, let us will set these by combining the init instruction with the setter instruction(s) client-side.